spec/integration/pgpapi.spec.js

```diff
@@ -0,0 +1,31 @@
+/*globals describe, beforeEach, require, expect, it */
+/*jslint indent:2,white:true,sloppy:true,node:true*/
```

Comment:
would prefer not having the ```white:true,sloppy:true``` and holding our selves to stricter style validation.